webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -53,16 +56,6 @@ class notificationContainer extends React.Component {
       })
     );
 
-    const translations = {
-      title: __('Notifications'),
-      unreadEvent: __('Unread Event'),
-      unreadEvents: __('Unread Events'),
-      emptyState: __('No Notifications Available'),
-      readAll: __('Mark All Read'),
-      clearAll: __('Clear All'),
-      deleteNotification: __('Hide this notification'),
-    };
```

Comment:
Where do all these strings come from now? I don't see any change sending them in the payload or otherwise.

Reply:
@tbrisker  They are coming from the default props of the DrawerWrapper.

https://github.com/theforeman/foreman/pull/6213/files/efe86855a5755640539f5308d8561f9c81d44db2#diff-b3b83bb5e440f68369f9ffd01095942dR127

Reply:
Which aren't extracted (since they come from patternfly react and we only extract strings in our code) - so there won't be any translations for them in the foreman translation files.

Reply:
Not sure I understand
`__('String')` Will get translated and extracted
`__(var)` Will only get translated?

Reply:
correct, to extract the strings (=make them available to the translators for translation), we scan all of our codebase for strings matching certain patterns (such as `_('string')`). The translators then translate the strings to the various languages (using a service called [transifex](https://www.transifex.com/foreman/foreman/foreman/)), and we pull in all of the translations right before every release.
the `_()` function family does a replacement of the content using a "dictionary" file - `$locale.mo` for ruby code, `$locale/app.js` for js - that maps the original strings to the translated ones.